submissions/kissik/port-sniffer/.gitignore

```diff
@@ -0,0 +1,2 @@
+/submissions/kissik/port-sniffer/node_modules
+/submissions/kissik/port-sniffer/package-lock.json
```

Comment:
The both path do start from the file system root. Therefore these particular exclusion rules just do not work. You may see that the project's `package-lock.json` is tracked despite the rules in the project's `.gitignore`.

The reason that your project's `node_modules` gets ignored is that the repo root `.gitignore` set the exclusion rule so it is applied throughout the entire file system sub-tree.

Same would work for your sub-project. Just keep this file empty or add a single line with `node_modules`. The latter will make your project consistent even if isolated into its own repo.